man/gnt-instance.rst

```diff
@@ -208,9 +207,9 @@ an instance with that name has already been added to the configuration
 as a forthcoming instance and the request is to replace this instance
 by the newly created real one.
 Note that if the reason for reserving an instance is that DNS names
-still need to be propagated, the reservation has to be done with
-``--no-name-check`` and ``--no-ip-check`` as these options are not
-implied by ``--forthcoming``.
+still need to be propagated, the reservation has to be done without
+``--name-check`` and ``--ip-check`` as these options are not implied
```

Comment:
The paragraph here doesn't make a lot of sense (the negation when they're negated already). I would just drop the "as these options are not implied by --forthcoming".